### ISSUE TITLE
New *Nullable bugpatterns not made available?

### ISSUE THREAD

**Issue by Stephan202** [google/error-prone#472]
The [release announcement](https://groups.google.com/forum/#!topic/error-prone-discuss/E2OXIVKOPJE) mentions three new suggestions related to nullability analysis, namely:

* `FieldMissingNullable`
* `ParameterNotNullable`
* `ReturnMissingNullable`

When I try use use any of these I get an error of the form:

> FieldMissingNullable is not a valid checker name

Running the following script in the `error-prone` source root, it seems the above checkers are the only ones not listed in `BuiltInCheckerSuppliers.java`:
```sh
#!/bin/sh

find ./core/src/main/java/com/google/errorprone/bugpatterns -name "*.java" \
  | xargs grep -l @BugPattern \
  | xargs -I{} basename '{}' .java \
  | while read C; do
      grep -q "$C" ./core/src/main/java/com/google/errorprone/scanner/BuiltInCheckerSuppliers.java \
        || echo "$C"
    done
```

**Comment by m-titov** [comment]
Please fix this bug

**Comment by eaftan** [comment]
We'd be happy to take a PR for this.  Those checks should go in the disabled section because they are intended as opt-in suggestions rather than errors or even warnings.

**Comment by m-titov** [comment]
Please approve pull request and release it as soon as possible. Thanks.

**Comment by m-titov** [comment]
Hi @eaftan When it could be released?

**Comment by Stephan202** [comment]
This was fixed version 2.1.2. Thanks!
